File: codigos/1 - estrutura sequencial/1012 - Area/Main.clj
(defn calcular-areas [a b c]
    (let [tri (format "%.3f" (/ (* a c) 2.0))
          circ (format "%.3f" (* 3.14159 (Math/pow c 2)))
          tra (format "%.3f" (/ (* (+ a b) c) 2.0))
          qua (format "%.3f" (float (Math/pow b 2)))
          ret (format "%.3f"(float (* a b)))]
        [tri circ tra qua ret]))

(defn printar-resultados [a b c]
    (let [areas (calcular-areas a b c)]
        (println "TRIANGULO:" (areas 0))
        (println "CIRCULO:" (areas 1))
        (println "TRAPEZIO:" (areas 2))
        (println "QUADRADO:" (areas 3))
        (println "RETANGULO:" (areas 4))))

(defn main []
    (apply printar-resultados (repeatedly 3 read)))

(main)

File: codigos/1 - estrutura sequencial/1012 - Area/Main.cc
// Online C compiler to run C program online
#include <stdio.h>
#include <math.h>
#define PI 3.14159
#define TRIANGULO(A,C) (A*C/2.0)
#define CIRCULO(C) (PI * pow(C,2))
#define TRAPEZIO(A,B,C) ((A+B)*C/2.0)
#define QUADRADO(B) (pow(B,2))
#define RETANGULO(A,B) (A*B)

int main() {
    double A,B,C;
    scanf("%lf %lf %lf", &A, &B, &C);
    printf("TRIANGULO: %.3lf\n", TRIANGULO(A,C)); 
    printf("CIRCULO: %.3lf\n", CIRCULO(C));
    printf("TRAPEZIO: %.3lf\n", TRAPEZIO(A,B,C));
    printf("QUADRADO: %.3lf\n", QUADRADO(B));
    printf("RETANGULO: %.3lf\n", RETANGULO (A,B));
}